app/controllers/application_controller.rb

```diff
@@ -7,6 +7,7 @@ class ApplicationController < ActionController::Base
   before_action :is_logged_in
   before_action :setup_query
   before_action :refresh_urs_if_needed, except: [:logout, :refresh_token]
+  before_filter :groups_enabled?
```

Comment:
Won't this redirect all the collections and drafts as well? Maybe this is better in the ManageCmrController? (also, if you move it, you may want to change what the ProviderHoldingsController inherits from)